ftw/simplelayout/browser/client/dist/simplelayout.css

```diff
@@ -445,50 +439,81 @@
   left: 0;
   z-index: 1;
 }
+.sl-toolbox .components .sl-toolbox-layouts a {
+  font-weight: bold;
+}
 .sl-toolbox .components a {
   position: relative;
   display: block;
-  padding: 10px 15px;
-  margin-bottom: -1px;
-  background-color: #fff;
-  border: 1px solid #ddd;
-  color: #555;
+  color: #7e7f84;
+  background: #2c2c34;
   width: 100%;
   box-sizing: border-box;
+  direction: rtl;
+  padding: 10px;
+  text-align: center;
+  border-bottom: 1px solid #30333a;
 }
 .sl-toolbox .components a.ui-draggable-dragging {
-  box-shadow: 0 0 70px 0 rgba(0, 0, 0, 0.75);
-  display: table;
-  width: auto !important;
+  box-shadow: 0 0 15px 0 rgba(0, 0, 0, 0.5);
+}
+.sl-toolbox .components a:before {
+  content: "";
+  display: inline-block;
+  height: 100%;
+  width: 2px;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: -2px;
 }
 .sl-toolbox .components a:hover {
   cursor: pointer;
-  color: #555;
+  color: #fff;
   text-decoration: none;
-  background-color: #f5f5f5;
+}
+.sl-toolbox .components a:hover:before {
+  background-color: #75ad0a;
+}
+.sl-toolbox .components a:hover > .description {
+  visibility: visible;
+  margin-right: 25px;
+  opacity: 1;
 }
 .sl-toolbox .components a i {
-  height: 17px;
-  width: 30px;
   display: inline-block;
   vertical-align: middle;
 }
+.sl-toolbox .components a .description {
+  white-space: nowrap;
+  text-align: right;
+  margin-right: 15px;
+  background-color: #2c2c34;
+  padding: 4px 7px;
+  color: #fff;
+  border-radius: 2px;
+  float: right;
+  margin-top: -2px;
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity .1s ease-in, margin-right .1s ease-out;
```

Comment:
I'd remove that. It feels much better without any transition. My two cents. :wink: